src/parser/jobs/war/modules/PrimalProcs.tsx

```diff
@@ -0,0 +1,70 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITY_MISSED_PROCS = {
+	1: SEVERITY.MAJOR,
+}
+
+export class PrimalProcs extends CoreProcs {
+
+	override trackedProcs = [
+		{
+			procStatus: this.data.statuses.PRIMAL_REND_READY,
+			consumeActions: [this.data.actions.PRIMAL_REND],
+		},
+		{
+			procStatus: this.data.statuses.PRIMAL_RUINATION_READY,
+			consumeActions: [this.data.actions.PRIMAL_RUINATION],
+		},
+		{
+			procStatus: this.data.statuses.WRATHFUL,
+			consumeActions: [this.data.actions.PRIMAL_WRATH],
+		},
+	]
+
+	protected override addJobSpecificSuggestions() {
+		const missedRend = this.getDropCountForStatus(this.data.statuses.PRIMAL_REND_READY.id)
+		const missedRuination = this.getDropCountForStatus(this.data.statuses.PRIMAL_RUINATION_READY.id)
+		const missedWrath = this.getDropCountForStatus(this.data.statuses.WRATHFUL.id)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PRIMAL_REND.icon,
+			content: <Trans id="war.primalrend.suggestions.dropped.content">
+				Using <DataLink action="INNER_RELEASE" /> grants <DataLink status="PRIMAL_REND_READY" />. Use it for a strong attack and to unlock the even stronger <DataLink action="PRIMAL_RUINATION" />.
+			</Trans>,
+			tiers: SEVERITY_MISSED_PROCS,
+			value: missedRend,
+			why: <Trans id="war.primalrend.suggestions.dropped.why">
+				You missed <Plural value={missedRend} one="# Primal Rend use" other="# Primal Rend uses" />.
+			</Trans>,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PRIMAL_RUINATION.icon,
+			content: <Trans id="war.primalruination.suggestions.dropped.content">
+				Using <DataLink action="PRIMAL_REND" /> grants you <DataLink status="PRIMAL_RUINATION_READY" /> for 20 seconds which lets you use <DataLink action="PRIMAL_RUINATION" />. It is your highest damage GCD, so try to spend it before it expires.
+			</Trans>,
+			tiers: SEVERITY_MISSED_PROCS,
+			value: missedRuination,
+			why: <Trans id="war.primalruination.suggestions.dropped.why">
+			You let <Plural value={missedRuination} one="# Primal Ruination proc" other="# Primal Ruination procs" /> time out.
```

Comment:
_Nit:_ This may just be a me thing, but I think it's nicer when similar suggestions in the same module use similar phrasing. Consider rewriting this and the one below to mirror the Primal Rend why text:
`You missed <Plural value={missedRuination} one="# use" other="# uses" /> of Primal Ruination.`
`You missed <Plural value={missedRwrath} one="# use" other="# uses" /> of Primal Wrath.`

Alternatively, you could rewrite the first one to mirror these two. Either way, the consistency would be nicer in my book. If you go that route, though, I would replace "time out" with "expire" so it mirrors the wording used in the content block.

Reply:
Yeah, thinking about it I agree that it's nicer if the phrasing matches. Since the "content" block already explains the player let the proc expire I will go with what you suggested and copy the Primal Rend why text.